### PR TITLE
Excluded VR code for Stadia platform support

### PR DESCRIPTION
**Pull request by Groovounet**
### Purpose of this PR
Excluded VR code for Stadia platform support

---
### Release Notes
Excluded VR code for Stadia platform support

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

Any test projects to go with this to help reviewers?

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=master&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?

**Halo Effect**: None, Low, Medium, High?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [7.0.0-preview] - 2019-XX-XX
 
+### Added
+- Excluded VR code for Stadia platform support
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/CHANGELOG.md]: Feels funny to have Added - Excluded. Maybe we should be in `changed` section instead?

**Comment by phi-lira** [comment]
@daves-unity don't know which one to poke from VR team to review this PR.

File: com.unity.render-pipelines.lightweight/CHANGELOG.md (hunk 1 above)
Comment (Groovounet) [reply com.unity.render-pipelines.lightweight/CHANGELOG.md]: Good point, I updated it to Initial Stadia support.

**Comment by Groovounet** [comment]
Replaced by https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/3715

**Pull request closed without merging**